docs/project/dogfooding.md

```diff
@@ -1,10 +1,8 @@
 # How to get up and running on .NET Core
 
-This document provides the steps necessary to consume a nightly build of
-.NET runtime and SDK.
+This document provides the steps necessary to consume a nightly build of .NET runtime and SDK.
 
-Please note that these steps are likely to change as we're simplifying
-this experience. Make sure to consult this document often.
+Please note that these steps are likely to change as we're simplifying this experience. Make sure to consult this document often.
 
 ## Obtaining nightly builds of NuGet packages
```

Comment:
Is the `https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet5/nuget/v3/index.json` feed still valid for current master?